build.sh

```diff
@@ -6,11 +6,31 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 PROJECT_DIR=$DIR/
 
 # This will: compile the project, run lint, run tests under JVM, package apk, check the code quality and run tests on the device/emulator.
-"$PROJECT_DIR"/gradlew --no-daemon clean
-"$PROJECT_DIR"/gradlew --no-daemon build -Dscan
-"$PROJECT_DIR"/gradlew --no-daemon test
-"$PROJECT_DIR"/gradlew --no-daemon connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.size=small
-if [ "$TRAVIS_BRANCH" == "development" ] || [ "$TRAVIS_BRANCH" == "master" ];
-then "$PROJECT_DIR"/gradlew --no-daemon connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.size=medium; fi
-if [ "$TRAVIS_BRANCH" == "master" ];
-then "$PROJECT_DIR"/gradlew --no-daemon connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.size=large; fi
\ No newline at end of file
+if [ "$TEST_SUITE" == "units" ];
+then
+"$PROJECT_DIR"/gradlew clean
+"$PROJECT_DIR"/gradlew build -Dscan
+"$PROJECT_DIR"/gradlew test; fi
+if [ "$TEST_SUITE" == "integration" ]
+then
+ android-wait-for-emulator
+ sleep 180
+ adb devices
+ adb shell input keyevent 82 &
+
+"$PROJECT_DIR"/gradlew connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.size=small;
```

Comment:
The small and large tests should execute in all suites fewer units:
Units suite should be executed always
Integration and integration_large should be executed in Pull Request and development branch but should not execute in master because the merge to master is during final release step, so how Integration and integration_large should be executed in the same time, I think that we can join it in one environment suite.